### PR TITLE
feat(billing): Add new data categories

### PR DESCRIPTION
**Pull request by scttcper**
https://getsentry.atlassian.net/browse/RV-1613

confused as to what to do with metrics and metric hours both using id 19?

### CODE REVIEW

File: static/app/constants/index.tsx [hunk 1]
```diff
@@ -326,6 +342,15 @@ export const DATA_CATEGORY_INFO = {
...
     uid: 19,
   },
+  // TODO: Deduplicate metrics and metricHour
+  [DataCategoryExact.METRIC_HOUR]: {
+    name: DataCategoryExact.METRIC_HOUR,
+    apiName: 'metrics',
+    plural: 'metrics',
+    displayName: 'metrics',
+    titleName: t('Metrics'),
+    uid: 19,
```
Comment (scttcper) [inline static/app/constants/index.tsx:352]: something will need to change here

**Review by isabellaenriquez — COMMENTED** [review]
do we need to keep `DataCategoryExact.METRICS` or can `DataCategoryExact.METRIC_HOUR` replace it? couldn't find places where it was used

File: static/app/constants/index.tsx [hunk 2]
```diff
@@ -318,6 +326,14 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Cron Monitors'),
     uid: 13,
   },
+  [DataCategoryExact.PROFILE_DURATION]: {
+    name: DataCategoryExact.PROFILE_DURATION,
+    apiName: 'profileDuration',
+    plural: 'profile durations',
+    displayName: 'profile duration',
+    titleName: t('Profile Durations'),
```
Comment (isabellaenriquez) [inline static/app/constants/index.tsx:334]: i think this should be profile seconds based on https://github.com/getsentry/getsentry/pull/13570

File: static/app/constants/index.tsx (hunk 1 above)
Comment (mbauer404) [reply static/app/constants/index.tsx]: @scttcper The data category is being changed from `DataCategory.METRIC_HOUR` to `DataCategory.METRIC_SECOND` to align with how the billing usage will be tracked. However, we will be displaying the usage to customers based on metric hours

File: static/app/constants/index.tsx [hunk 3]
```diff
@@ -318,6 +326,14 @@ export const DATA_CATEGORY_INFO = {
...
+    name: DataCategoryExact.PROFILE_DURATION,
+    apiName: 'profileSeconds',
+    plural: 'profileSeconds',
+    displayName: 'profile seconds',
+    titleName: t('Profile Seconds'),
+    uid: 17,
+  },
   [DataCategoryExact.METRIC_SECOND]: {
     name: DataCategoryExact.METRIC_SECOND,
     apiName: 'metricSecond',
```
Comment (mbauer404) [inline static/app/constants/index.tsx:339]: @scttcper there were some adjustments to the metrics category - would you be able to add those updates to this PR? 

```
[DataCategoryExact.METRIC_SECOND]: {
    name: DataCategoryExact.METRIC_SECOND,
    apiName: 'metricSeconds',
    plural: 'metric hours',
    displayName: 'metric hours',
    titleName: t('Metrics Hours'),
    uid: 19,
  },
  ```

**Review by mbauer404 — APPROVED** [review]

**Review by isabellaenriquez — COMMENTED** [review]
sorry one more thing lol

**Review by isabellaenriquez — APPROVED** [review]

**Pull request merged**